theories/_CoqProject

```diff
@@ -1 +1,3 @@
 -Q . Stdlib
+# introduced in Rocq 9.2
+-arg -w -arg +level-tolerance
```

Comment:
Why is this warning special?

Reply:
My preference is to use the same flags for all build methods, primarily to avoid spending dev time on troubleshooting inconsistencies. I do sometimes disable warnings-as-errors during development, but I think it is good to have by default. I don't see the case for having it for just one warning in one build method but for all warnings in another. Previous discussion:

- https://github.com/rocq-prover/rocq/pull/16950